p2p/les.py

```diff
@@ -314,8 +315,12 @@ def send_get_block_bodies(self, block_hashes: List[bytes], request_id: int) -> N
         header, body = GetBlockBodies(self.cmd_id_offset).encode(data)
         self.send(header, body)
 
-    def send_get_block_headers(self, block_number_or_hash: Union[int, bytes],
-                               max_headers: int, request_id: int, reverse: bool = True
+    def send_get_block_headers(self,
+                               request_id: int,
```

Comment:
I don't care too much about having `request_id` as the first or last argument, but all `LESProtocol` methods have it as the last, so if you feel strong about having it as the first, we should change the others as well, to be consistent

Reply:
:+1: I'd like it to be consistently ordered.  Last works for me.